docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_http_filters_cache:
+
+HTTP Cache Filter
+=================
+**Work in Progress--not ready for deployment**
+
+HTTP caching can improve system throughput, latency, and network/backend load
+levels when the same content is requested multiple times. Caching is
+particularly valuable for edge proxies and browser-based traffic, which
+typically include many cacheable static resources, but it can be useful any time
+there is enough repeatedly served cacheable content.
+
+Configuration
+-------------
+* :ref:`Configuration API <envoy_api_msg_config.filter.http.cache.v2.cacheConfig>`
+* This filter should be configured with the name *envoy.filters.http.cache*.
+
+The only required configuration field is :ref:`name
+<envoy_api_field_config.filter.http.cache.v2.CacheConfig.name>`, which must
+specify a valid cache storage implementation linked into your Envoy
+binary. Specifying 'envoy.extensions.http.cache.simple' will select a proof-of-concept
+implementation included in the Envoy source. More implementations can (and will)
+be provided by implementing Envoy::Extensions::HttpFilters::Cache::HttpCache. To
+write a cache storage implementation, see `Writing Cache Filter
+Implementations <https://github.com/envoyproxy/envoy/blob/master/source/docs/cache_filter_plugins.md>`_
```

Comment:
use a `:repo:` link here to lock the file version.

Reply:
done